doc/apiv3-documentation.apib

```diff
@@ -1608,7 +1608,6 @@ Note that regular expressions used in the API follow the rules of [Ruby Regular
     + Body
 
             {
-                "_type": "Schema",
```

Comment:
I'd like to update the documentation as soon as we have the real solution to be honest...

As nobody seems to be interested in requiring the `_type` for `PATCH` requests I'd say lets change that now:

1. I reintroduce the `"_type": "Schema"`
2. I remove `_type` attributes from schema listings
3. Specification is updated to state that the `_type` is not required for `PATCH`ing

Any objections?